app/services/allocations/finder.rb

```diff
@@ -73,6 +73,8 @@ def apply_date_range_filters(scope)
     end
 
     def split_params(name)
+      return if filter_params[name].blank?
+
       filter_params[name]&.split(',')
```

Comment:
You could remove the `&` here now as it's guarded by the new line above.

```suggestion
      filter_params[name].split(',')
```